src/servers/graphql-server.ts

```diff
@@ -114,7 +115,10 @@ const sessionContext = ({
         account = loggedInAccount
       }
 
-      addAttributesToCurrentSpan({ [ENDUSER_ALIAS]: domainAccount?.username })
+      addAttributesToCurrentSpan({
```

Comment:
if we use here `addAttributesToCurrentSpanAndPropagate` would avoid the implementation in the resolvers?

Reply:
IIRC @bodymindarts told me this would not work (didn't try myself) because the `context` function doesn't launch the resolvers, it's more like a:
```
context() {}
resolver() {}
``` 

than 
```
context() {
...
...
...
... resolver() {}
}
```

Reply:
That is correct it won't work like @nicolasburtey says